ios/platform-features/auto-layout/programmatical-constraints.workbook

```diff
@@ -0,0 +1,120 @@
+---
+uti: com.xamarin.workbook
+platforms:
+- iOS
+---
+
+### Programmatically Creating Constraints
+
+The goal of this workbook is to explain **advanced auto layout** concepts by showing how to create constraints **programmatically**.
+
+*Whenever possible, we recommend you use the iOS designer to create your contraints because you can easily visualize, edit, manage and debug them.*
+
+```csharp
+var orangeLabel = new UILabel () {
+        Text = "Orange",
+        TextColor = UIColor.White,
+        TextAlignment = UITextAlignment.Center,
+        BackgroundColor = UIColor.Orange
+};
+
+// This could be any parent view you add UI elements to.
+var rootView = RootViewController.View;
+rootView.AddSubview (orangeLabel);
+```
+
+#### Absolute coordinates
+
+Let’s position the `orangeLabel` at the top left corner manualy first so we can see it.
+
+```csharp
+orangeLabel.Frame = new CGRect (20, 20, 120, 50);
+```
+
+If we wanted to center this label in the middle of the screen we could, *technically*, do it using **absolute** x,y coordinates but this is **not** the most optimal, responsive and simple (will imply quite a bit of math) way to do it.
+
+#### Begin with Constaints
+
+`TranslatesAutoresizingMaskIntoConstraints` is **key** to using manual constraints.
+
+By default this is set to `true` which lets the system create a set of constraints for you.
+
+However it prevents you from adding **additional constraints**, see API doc:
```

Comment:
I'd probably alter the sentence a little to say

"However, using `TranslatesAutoresizingMaskIntoConstraints` prevents you from adding **additional constraints**, as described in [Specific API doc name] doc:"